python/py-pydns/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+set real_name       pydns
+name                py-${real_name}
+version             2.3.6
+revision            0
+worksrcdir          ${real_name}-${version}
+distfiles           ${real_name}-${version}${extract.suffix}
+categories-append   security
+license             Python
+maintainers         @BjarneDMat openmaintainer
+description         Python 2 DNS library
+long_description    ${description}
+platforms           darwin
+homepage            http://pydns.sourceforge.net/
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+
+python.versions     27
```

Comment:
Again can python 37 be used here instead?

Reply:
no. this is a 2.7 only port. the 3.7 port is py_py3dns